includes/class-wc-taxjar-integration.php

```diff
@@ -459,13 +459,24 @@ public function create_or_update_tax_rate( $location, $rate, $tax_class = '', $f
 			'tax_rate_class' => $tax_class,
 		);
 
-		$wc_rate = WC_Tax::find_rates( array(
-			'country' => $location['to_country'],
-			'state' => $location['to_state'],
-			'postcode' => $location['to_zip'],
-			'city' => $location['to_city'],
-			'tax_class' => $tax_class,
-		) );
+		if ( version_compare( WC()->version, '3.2.0', '<' ) ) {
+			$wc_rate = WC_Tax::find_rates( array(
+				'country' => $location['to_country'],
+				'state' => $location['to_state'],
+				'postcode' => $location['to_zip'],
+				'city' => $location['to_city'],
+				'tax_class' => $tax_class,
+			) );
+        } else {
+			$wc_rate = WC_Tax::find_rates( array(
+				'country' => $location['to_country'],
+				'state' => sanitize_key( $location['to_state'] ),
```

Comment:
If `sanitize_key` is the only difference here, can we DRY up this code?

Reply:
Resolved in most recent commit